cypress/e2e/0-platformless/1-cleanup.spec.ts

```diff
@@ -1,14 +1,15 @@
-/// <reference types="cypress" />
-
 before(() => {
   cy.disconnectMetamaskWalletFromAllDapps()
 })
 
 describe("post-test cleanup", () => {
   before(() => {
-    cy.wait(11_000)
-    cy.visit(`/${Cypress.env("guildUrlName")}`, { failOnStatusCode: false })
-    cy.visit(`/${Cypress.env("guildUrlName")}`, { failOnStatusCode: false })
+    cy.visit(
+      `/${Cypress.env("platformlessGuildUrlName")}-${Cypress.env("DEPLOYMENT_ID")}`,
+      {
+        failOnStatusCode: false,
+      }
+    )
   })
 
   it("cleans up test guild", () => {
```

Comment:
I think I had something similar when I was doing tests, but can't remember exactly why it is needed to check for 404/client-side error here, is this still required? If so, in case of 404 does the guild get cleaned up?
I think there used to be some backend inconsistency (or maybe something related to FE static generation(?))

Reply:
yes, I believe you added it because sometimes the frontend will still show a 404 page, and then revalidates the guild on client-side 👍